### ISSUE TITLE
Log scale saving increments

### ISSUE THREAD

**Issue by RollaTroll** [llbit/chunky#347]
Add an option to save render dump and image at 2^n SPP intervals. In such a way, each step provides roughly equal visually perceived improvements.
A sampling level of 0 is a SPP of 1
1 = 2
2 = 4
3 = 8
4 = 16
5 = 32
Etc.

**Comment by empirebuilder1** [comment]
However compute time would also increase logarithmically and probably would not be desirable for most users. The SPP amount required to get a good result also varies greatly depending on render settings, and no one scale could possibly account for all (or even most) possible settings.

**Comment by RollaTroll** [comment]
That is true. However, one could just as easily say the visual improvements from one SPP to the next decrease in the same way. Ultimately, the end result will be the same. However, an option to only save snapshots at 2^n would provide a nice alternative to the current mode, though it should by no means be forced on every user.

**Comment by RollaTroll** [comment]
![series](https://cloud.githubusercontent.com/assets/11786132/12863408/f26f21bc-ccc0-11e5-8b8f-a983215a4150.png)
Here is an example I did

**Comment by empirebuilder1** [comment]
Interesting, it seems to work pretty well. Perhaps chunky needs more slider bars?

**Comment by RollaTroll** [comment]
I think it could be implemented as an option in "Save dump once every ... frames". If possible, it might be nice to give an option for any x^n option, but that would be of minimal benefit.

**Comment by llbit** [comment]
One of the reasons for the render dump frequency being based on constant frame intervals is to avoid losing lots of progress if Chunky or the computer running Chunky crashes. It does clog up the render folder with lots of pictures though, so that's why the snapshots can be disabled when saving a render dump.

This feature request seems like it would add very little utility in itself, so I'm not too keen on implementing it. It is still an interesting suggestion.

Perhaps this feature would be better suited as a plugin.

**Comment by W-Floyd** [comment]
That's a fair point. I understand.
Perhaps if the dump could be saved at the regular frequently, and snapshots (and by extension, dump as well) at a second frequency? I have a feeling it would get awfully messy to set up though :/
